autolab/Autolab
# Lint views/doorkeeper and remove more unused files

## Summary

### Summary generated by Reviewpad on 12 Sep 23 21:23 UTC
This pull request includes linting changes for the doorkeeper application views and the removal of unused files. The changes include fixing form tag attributes, updating link attributes, and removing two unused files.


## Description

- lint `views/doorkeeper` using `erblint`
- remove unused files in `lib/views`

## Motivation and Context


- As part of summer linting

## How Has This Been Tested?



- ensure that doorkeeper views still work, create/edit/destroy oauth apps (Go to "Manage API Applications" in "Manage Autolab")
- ensure that authorization pages work (Go to "Manage API Applications" and authorize an application, go to "User Profile" and click "Manage Authorized Applications")

## Checklist:
